bitbots_body_behavior/config/body_behavior.yaml

```diff
@@ -1,246 +1,254 @@
-
-    # Time to wait in ready state before moving to role position to give the localization time to converge.
-    ready_wait_time: 4
-
-    # When the ball has not been seen for `ball_lost_time` seconds,
-    # it is considered lost and will be searched
-    ball_lost_time: 8
-
-    # The orientation threshold defining which range (in Degrees) is acceptable as aligned to the goal (in each direction)
-    goal_alignment_orientation_threshold: 20
-
-    # When the goal has not been seen for `goal_lost_time` seconds,
-    # it is considered lost and will be searched
-    goal_lost_time: 30
-
-
-    # When the ball is closer than `ball_close_distance` meters
-    # it is in a reachable area of the robot
-    ball_close_distance: 1.5
-
-    # the maximal allowed standard deviation of the ball twist.
-    ball_twist_precision_threshold:
-      x_sdev: 0.3
-      y_sdev: 0.3
-
-    # the duration after which a ball_twist is considered irrelevant.
-    ball_twist_lost_time: 2
-
-    # the maximal allowed standard deviation of the ball position.
-    ball_position_precision_threshold:
-      x_sdev: 0.5
-      y_sdev: 0.5
-
-    # An area in which the ball can be kicked
-    # defined by min/max x/y values in meters which represent ball positions relative to base_footprint
-    # http://www.ros.org/reps/rep-0103.html#axis-orientation
-    kick_x_enter: 0.24
-    kick_x_leave: 0.30
-    kick_y_enter: 0.14
-    kick_y_leave: 0.16
-
-    # defines the radius around the goal (in form of a box)
-    # in this area, the goalie will react to the ball.
-    # the radius is the margin around the goal to both y and the positive x directions
-    ball_dangerous_goal_radius: 2
-
-    # defines the area in which the goalie will not attempt to fall in front of the ball
-    # this value represents the y displacement of the ball relative to the center of the robot
-    ball_dangerous_center: 0.1
-
-    # The defensive area is an area in which the players behave more defensive then usual
-    # (defensive players are actively going to the ball and goalies move in the goal to be able to block the ball).
-    # This affects the BallInDefensiveArea decision.
-    # The area is described as portion of the field [0, 1] always measured from the own goal.
-    # A value of 0.25 means, that the quarter in front of the own goal is the defensive area.
-    defensive_area: 0.5
-
-    # This is the offset the goalie keeps to avoid crashes with poles when blocking balls.
-    # The value describes the offset in meters from the goal line.
-    block_position_goal_offset: 0.15
-
-    # this factor defines how extreme the goalie reacts to a ball offset
-    block_position_gradient_factor: 4
-
-    # configurations for the use of bitbots_dynamic_kick package
-    dynamic_kick:
-      # time to wait for a dynamic_kick server
-      wait_time: 10
-      # base topic under which an actionserver listens for KickAction messages
-      topic: "dynamic_kick"
+/**:
```

Comment:
no it still only loads these parameters on the corresponding node on which these parameters are loaded in the launch file
it is necessary to not specify the body behavior here explicitly since we load these parameters also on the head behavior node